### ISSUE TITLE
Include SQLite.swift in the main pods repo

### ISSUE THREAD

**Issue by cadizjavier** [stephencelis/SQLite.swift#257]
It seems like the only way to use this library with cocoapods is checking out the github repo.
Is there any particular reason to that ?
I would like to see this library in the main pods repo.

![cocoapods](https://cloud.githubusercontent.com/assets/1702835/10592357/98727264-7689-11e5-98cf-9d2b23639f63.jpg)

**Comment by stephencelis** [comment]
It's on the roadmap! Just wanted to fix up a few things in the interface a bit.

**Comment by cadizjavier** [comment]
Glad to know. Thanks

**Comment by rpowelll** [comment]
Any progress on this? I want to include SQLite.swift as a dependency of my project and the lack of a Podspec in the central repo's preventing that.

**Comment by stephencelis** [comment]
@rpowelll I'll try to get a version out this weekend. Probably won't be a final 1.0.0, but a 0.9.x should alleviate this requirement.

**Comment by rpowelll** [comment]
Thanks that'd be great

**Comment by cjwirth** [comment]
Just wondering how this is coming along. Looking forward to it :+1:

**Comment by warn4n** [comment]
+1 for prioritizing this. It's a bit disconcerting to get breaking API changes any time we run "pod install"! Love this library!

**Comment by stephencelis** [comment]
Agh. @warn4n sorry about that. The API is definitely in flux but it should be much easier to lock in. We're close to a CocoaPods release. I just haven't had the free time I'd hoped for!

**Comment by warn4n** [comment]
Awesome news, thanks so much!

**Comment by mikemee** [comment]
Big progress towards this, I hope!  After a full day of banging my head against all sorts of interesting and less interesting things, a PR for this https://github.com/stephencelis/SQLite.swift/pull/314. 

If you're inclined, please modify your Pod file to try:
```
pod 'SQLite.swift',
  git: 'https://github.com/stephencelis/SQLite.swift.git', :branch => 'publicpod'
```
No API changes, just smoothing some packaging edges, I hope, for a first public pod release. I'm a CocoaPods newbie, so although I've checked this every way I can think of, there may be a remaining hiccup or two (https://github.com/CocoaPods/CocoaPods/issues/4607 didn't help).

If any CocoaPods experts out there have an idea on how to fix the seemingly erroneous warning (`Umbrella header for module 'SQLite' does not include header 'fts3_tokenizer.h`), any help is much appreciated. 

_I found there is a lot of uncharted territory in the "new Swift Framework based on non-Swift `sqlite3` C library" realm, and that CocoaPods tends to trip over itself a little (Xcode is in flux here too). So, in an effort to keep tweaking to a minimum, I gave up trying to remove this warning for now. There's already an extraneous `module.modulemap` added back into the build because I gave up trying to coax CocoaPods to generate a valid one and/or to put it in the right place...  The combination of my newness to CocoaPads, the `SQLite.swift` package name to `SQLite` framework name, plus the (new) `module.modulemap` file for the C `sqlite3` library (per latest Xcode standards & forthcoming Swift Package Manager), all seems to create an interesting puzzle. In the name of pragmatism, I quit once I had a working build with minimal magic intervention Here's hoping it all works for you too...!._

**Comment by stephencelis** [comment]
:tada: https://cocoapods.org/pods/SQLite.swift

Thanks go to @mikemee for all his help!

**Comment by Ryannnnnnn** [comment]
:tada: it works! (though the umbrella warning is annoying)

**Comment by mikemee** [comment]
> though the umbrella warning is annoying

Indeed!  As I said in the PR #314 code review thread:

> I tried all sorts of things. CocoaPods is a magical mess was the only conclusion I could come up with. I essentially had to choose between keeping Xcode happy, keeping CocoaPods happy, or crawling under the bed to cry. I hoped that once this PR went "live", someone would point out the stupidity of my ways and offer a fix!

TL;DR: PR's to fix this are solicited! 😎
